examples/pytorch/nlp/huggingface_models/text-classification/pruning/eager/run_glue_no_trainer.py

```diff
@@ -505,7 +506,7 @@ def preprocess_function(examples):
             "pruning_type": "snip_momentum",
             "pruning_scope": "global",
             "sparsity_decay_type": "exp",
-            "excluded_op_names": ["classifier", "pooler", ".*embeddings*"],
+            "excluded_op_names": ["pooler", ".*embeddings*"],
```

Comment:
@WeiweiZhang1  could we try to remove pooler layer  when we test block criterion

Reply:
The existing sparse criteria does not support pooler sparsity because pooler weights have no gradient.